### ISSUE TITLE
Fix the logo in the react icons README.md

### ISSUE THREAD

**Issue by kartikjoshi267** [Bookingjini-Labs/bookingjini-icons#400]
### Describe the bug

The logo in the react icons `README.md` is not showing up.

### Expected behaviour

Ideally, there should be a logo instead of the alt text.

### Screenshots

![image](https://github.com/Bookingjini-Labs/bookingjini-icons/assets/96565730/2c2dd859-0407-4d2c-92ca-656e4bf3c6e8)

### Additional context

This issue has been opened as discussed in PR #289 and please assign this to me as a part of GSSOC23

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by AshutoshDash1999** [comment]
@kartikjoshi267 assigning this to you
